client/cli/ok.py

```diff
@@ -92,6 +93,12 @@ def parse_input(command_input=None):
 
 def main():
     """Run all relevant aspects of ok.py."""
+
+    #Checking user's Python bit version
```

Comment:
Please move this block below `log.debug(args)`. All log messages should be made after we set the log level.